### PR TITLE
Jellof rice recipe

### PR DESCRIPTION
**Pull request by Fidelta**
I create three recipes of Jellof rice, Beans porridge and Egg sauce. I also add the images of these three dishes and created links to their pages.

### CODE REVIEW

File: Recipes/Beans.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+~<!DOCTYPE html>
```
Comment (antoHero) [inline Recipes/Beans.html]: this symbol ~ on line 1 shows on your page and you do not need it

**Review by antoHero — CHANGES_REQUESTED** [review]
Good job so far Precious 🔥, I am proud to see your first project. I noticed a few things you need to fix
- when naming your files, you cannot have spaces (eg egg sauce.html). I suggest you call the file eggsauce.html, egg_sauce.html, or egg-sauce.html. Do the same with jellof rice and your images too. After renaming the files, remember to update the links in index.html
- the image on jellof rice recipe page is broken (it doesn't show) because you added a dot when you were linking it on the page. Kindly check and fix that as well
- your images can only have one extension eg .jpg instead of .jpg.jpg you used. Could you remove the extra .jpg and you are good to go? Don't forget to update the way you are linking the images on each recipe page
- finally, you have 3 recipe pages but only 2 are active, abi you no want make we learn how to cook egg? 😉

A few modifications left and your first project will be live 🚀🚀

**Review by Kadunapikin — CHANGES_REQUESTED** [review]
Good job Precious, you did well on your first project!  However, make necessary correction on line 1 of Beans.html and also rename your html pages using the Camel casing naming convention.

**Comment by antoHero** [comment]
Congratulations @Fidelta on finishing your first project. We are happy to see you are making progress and have basic  knowledge in
- HTML, and
- git

Before we/you merge your work, consider renaming your images and HTML files using camel case as suggested by @Kadunapikin. Start renaming your folders also with a small case

`eg rename BeansPorridge.jpg to beansPorridge.jpg`

**Review by antoHero — APPROVED** [review]

**Review by Kadunapikin — APPROVED** [review]
Great job and congratulations @Fidelta  on completing your first project! You can go ahead and now merge. Happy coding!!!

**Comment by Fidelta** [comment]
Well, it's my first project ever and I am excited practicing as I am learning. I wont give up until i get to the desired destination.

**Pull request merged**